### PR TITLE
Fix UG Bugfixes Part 5

### LINKED ISSUE #139: Modify Staffsync GUI Part 2
Continuation of #111 for v1.5

### LINKED ISSUE #154: Update UG for 2101
Refer to peer review comments here: https://docs.google.com/document/d/1WcBkk7E-YvEeoHcUilXl1Jb8eKjKTm6fE1MLQQQTYzw/edit?tab=t.0

Issues remaining:

- Use of English should be changed to more user-centric

### PR DESCRIPTION
**Pull request by InfinityTwo**
## Describe your changes
Modify dropdowns to be enabled by default to support pdf
Add boxes for format
Modify all commands section format for consistency
Fix hyperlinks issue caught by peer review
Modify somestyling throughout UG
Modify images to be slightly reduced in size
Add a warning box for clear command in UG that there is no confirmation

image
image
image
image

## Issue ticket number and link
Continuation of #153
Partially fixes issues found in #154 
Closes #139

## Checklist before requesting a review
- [x] I have performed a self-review of my code.
- [x] If it is a core feature, I have added thorough tests.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -334,23 +359,43 @@
...
+  * `e` for employees
+  * `ph` for potential hires.
+
+  `INDEX`: The index number shown in the displayed person list.
+  * `INDEX` **must be a positive integer** 1, 2, 3, …​
+  * `INDEX` **must be within the size of the list** shown.
 
   Examples:
-  * `list` followed by `delete e 2` deletes the 2nd person in the employee list.
+  * `list` followed by `delete ph 2` deletes the 2nd person in the potential hire list.
```
Comment (LimErnest) [inline docs/UserGuide.md]: list is no longer a valid command, I think should change to list ph instead

File: docs/UserGuide.md [hunk 2]
```diff
@@ -334,23 +359,43 @@
...
   * `find e Betsy` followed by `delete e 1` deletes the 1st employee in the results of the `find` command.
```
Comment (LimErnest) [inline docs/UserGuide.md]: find command also have changed

File: docs/UserGuide.md [hunk 3]
```diff
@@ -514,25 +597,26 @@ and with phone number either `12345678` or `87654321`.
   email either `john@example.com` or `alice@example.com`
   * `find ph d/IT r/SWE Manager` returns potential hires with department `IT`, and role either `SWE` or `Manager`
 
-Example: "find ph n/John".
-![result for 'find ph n/John'](images/findjohnResult.png)
+  Example: "find ph n/John".
+  ![result for 'find ph n/John'](images/findjohnResult.png)
```
Comment (LimErnest) [inline docs/UserGuide.md:601]: i think the john here needs to be capital J like John for the images/findjohnResult

**Review by LimErnest — COMMENTED** [review]
Just some minor changes

File: docs/UserGuide.md (hunk 3 above)
Comment (InfinityTwo) [reply docs/UserGuide.md:601]: Good find! However, that issue will be resolved in #203 which has yet to be merged

**Review by LimErnest — APPROVED** [review]
LGTM!

**Pull request merged**